### DISCUSSION TITLE
使用Instruct系列，出现重复回答的问题，不知道是模型就这样还是我们接口调用的问题。

### DISCUSSION THREAD

**Discussion by kingemperorcz** [ymcui/Chinese-LLaMA-Alpaca-3#66] [Q&A]
![微信图片_20240606130606](https://github.com/ymcui/Chinese-LLaMA-Alpaca-3/assets/137819598/cc6fcb45-e916-4201-ab20-c0057c903f26)就问了一个问题

**Comment by ymcui** [comment]
不清楚你怎么调用的，要么是没套指令模板要么就是参数设置有问题。
正常来说一般不会出现这种情况。你可以试试在线demo：https://huggingface.co/spaces/hfl-rc/llama-3-chinese-8b-instruct-demo
